src/components/board/board.test.tsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
```

Comment:
Another humble request in addition to the ones here: can you work at renaming the repo?  It can be whatever you want, but it should indicate this is for TTT React or TTT front-end...you usually don't see TypeScript mentioned in a repo name